pkgs/data/fonts/nerdfonts-patcher/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv
+, fetchFromGitHub
+, lib
+}:
+
+stdenv.mkDerivation rec {
```

Comment:
Can you try to use buildPythonApplication instead of mkDerivation as described in https://nixos.org/manual/nixpkgs/stable/#python 15.19.2.2.2. buildPythonApplication function.